### PR TITLE
feat: update service runner from multi-geth to core-geth

### PR DESCRIPTION
**Pull request by zcstarr**

### CODE REVIEW

File: src/service-runner-config.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
             "start": [
               "--classic",
               "--datadir",
-              "${SERVICE_DIR}/multi-geth/1.9.9/classic",
+              "${SERVICE_DIR}/core-geth/1.11.2/classic",
```
Comment (meowsbits) [inline src/service-runner-config.json]: Same question as above about using a new dir and leaving one disused.

Comment (zcstarr) [reply src/service-runner-config.json]: It does leave a dangling path. There's always been a discussion of garbage collection I believe Zack put up an issue some time ago. We have the information to garbage collect, but it's uncertain if we should leave that in the hands of the user.  Service runner atm doesn't allow for removal without access.

I'm probably going to punt this issue until we have a better story around shared config . Atm the deploy step will blast this data.

Comment (zcstarr) [reply src/service-runner-config.json]: The path to garbage collection is in the manifest.json file

**Comment by zcstarr** [comment]
> https://github.com/etclabscore/jade-service-runner/tree/core-geth
> 
> ```
> commit 55ffcc7bd34f78bee3d8ffcbd9e59256e9bbd92e
> Author: meows <b5c6@protonmail.com>
> Date:   Fri Mar 20 10:53:37 2020 -0500
> 
>     fix(core-geth examples): bump version from v1.9.0->v1.11.2
>     
>     These are just demos and comments, but it's nice
>     to have them up-to-date, or at least
>     to have them having been at least once upon a time
>     working.
>     
>     Signed-off-by: meows <b5c6@protonmail.com>
> 
> diff --git a/client.ts b/client.ts
> index 6060e0c..87c0ccd 100644
> --- a/client.ts
> +++ b/client.ts
> @@ -1,10 +1,10 @@
>  /*const ServiceRunner = require('./build/client/typescript/build/index').default;
>  const client = new ServiceRunner({ transport: { type: "http", port: 8002, host: "localhost" } });
> -client.installService("core-geth", "1.9.0")
> +client.installService("core-geth", "1.11.2")
>    .then(() => client.listInstalledServices())
>    .then(() => client.listRunningServices())
>    .then(console.log)
> -  .then(() => client.startService("core-geth", "1.9.0", "mainnet"))
> +  .then(() => client.startService("core-geth", "1.11.2", "mainnet"))
>    .then(console.log)
>    .then(() => client.listRunningServices())
>    .then(console.log)
> diff --git a/examples/index.ts b/examples/index.ts
> index 59fe786..da2add1 100644
> --- a/examples/index.ts
> +++ b/examples/index.ts
> @@ -13,20 +13,20 @@ const ethClient = new EthClient({
>      host: "localhost",
>      port: 8002,
>      type: "http",
> -    path: "core-geth/mainnet/1.9.0",
> +    path: "core-geth/mainnet/1.11.2",
>    },
>  });
>  
>  console.log("starting client");
> -client.installService("core-geth", "1.9.0")
> +client.installService("core-geth", "1.11.2")
>    .then(() => client.listInstalledServices())
>    .then(() => client.listRunningServices())
>    .then(console.log) //tslint:disable-line
> -  .then(() => client.startService("core-geth", "1.9.0", "mainnet"))
> +  .then(() => client.startService("core-geth", "1.11.2", "mainnet"))
>    .then(console.log)//tslint:disable-line
>    .then(() => client.listRunningServices())
>    .then(console.log)//tslint:disable-line
> - client.startService("core-geth", "1.9.0", "mainnet")
> + client.startService("core-geth", "1.11.2", "mainnet")
>    .then(() => ethClient.eth_getBalance("0xc1912fee45d61c87cc5ea59dae31190fffff232d", "0x0"))
>    .then(console.log)
>    .catch((e) => {
> ```

Yeah this would be the way, but in reality it's deprecated. Signatory has the pattern that should be used for this and could use some TLC. I have a wave of updates coming soon to address this. I'd consider this to be a bit out of the scope esp. because it's not a pattern someone should follow. 

I think i'm a bit anxious to land this one change and then move forward with the refactorings to address these issues.

**Comment by zcstarr** [comment]
@meowsbits PTAL when you get chance.

**Pull request merged**